src/kagglehub/models.py

```diff
@@ -38,7 +39,7 @@ def model_upload(
     """
     # parse slug
     h = parse_model_handle(handle)
-
+    logger.info(f"Uploading Model {handle}")
```

Comment:
Let's use `handle.to_url()`. This will let users easily identify / navigate to the detail page for the model they are uploading.

nit: Add `...` at the end given this action is in process.

Reply:
done